api/server/server.go

```diff
@@ -250,7 +254,7 @@ func NewServer(conf Config) (*Server, error) {
 	if conf.Devnet {
 		conf.FFSMinimumPieceSize = 0
 	}
-	cs := filcold.New(ms, dm, ipfs, chain, l, conf.FFSMinimumPieceSize)
+	cs := filcold.New(ms, dm, ipfs, chain, l, lsm, conf.FFSMinimumPieceSize, conf.FFSMaxParallelDealPreparing)
```

Comment:
This new component is used by Cold Storage, plus a new flag that indicates how many data-intensive tasks we throw at lotus in parallel. (More on this later).